pubspec.yaml

```diff
@@ -33,6 +33,7 @@ dev_dependencies:
   pedantic: ^1.4.0
   test: ^1.5.1
   meta: ^1.1.7
+  react: ^4.6.1
```

Comment:
technically, we don't needs this, we could fake it instead since our codemod doesn't actually look to see if we are using react.dart ill show you a thought in next code